src/client.rs

```diff
@@ -153,6 +153,14 @@ pub struct RequestBuilder {
 }
 
 impl RequestBuilder {
+
+    /// Get the url result.
+    ///
+    /// Will contain an error if the given URL was invalid.
+    pub fn get_url(&self) -> &Result<Url, ::UrlError> {
```

Comment:
Oh right, I forgot about this. Hm, exposing that it is currently stored as a `Result` is unfortunate. Thoughts in review-comment.